### DISCUSSION TITLE
<Span> tag ?

### DISCUSSION THREAD

**Discussion by Ranjankumar666** [chakra-ui/chakra-ui#3870] [Q&A] [answered]
Is there something like jsx <span> tag to make small changes in between elements?

**Comment by chr-ge** [comment, marked as answer]
You can use `chakra.span` to make a Chakra span element. [chakra-factory](https://chakra-ui.com/docs/features/chakra-factory)

**Comment by Ranjankumar666** [comment]
Yeah i just found it today in thw documentation about chakra factory
functions. Thanks!❤️
